### PR TITLE
Update license and keyword at lib Jq plot

### PR DESCRIPTION
**Pull request by programmiri**
Pull request for issue: #11931

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Squashing into only one commit now.

File: ajax/libs/jqPlot/package.json [hunk 1]
```diff
@@ -8,15 +8,11 @@
...
-    "MIT",
-    {
-      "type": "GPL",
-      "url": "http://www.opensource.org/licenses/gpl-license.php"
-    }
+    "popular",
+    "plotting",
+    "graphing"
   ],
+  "license": "(MIT AND GPL-2.0)",
```
Comment (maruilian11) [inline ajax/libs/jqPlot/package.json]: Let's use
```
"licenses": [
  "MIT",
  "GPL-2.0"
],
```

**Comment by maruilian11** [comment]
I think we also need `CC-BY-3.0`. https://github.com/jqPlot/jqPlot/blob/7a8a8ea7d2d0d24a63426b539368a1ca250fe103/README.md#date-instance-methods

**Comment by maruilian11** [comment]
@programmiri You did a great job! Since you are not a one-time contributor, I think you can try to squash all commits into a clear one. You can use `git rebase -i HEAD~2`, it will show the last two commits you have done. And modify the last commit(the latest one) the word `pick` in the head of line to `f`. Then, save and leave. It should help you squash these two commits. Remember to rebase with our latest master branch. 👍  
Feel free to ask any question, we are so glad that there is a new contributor. 😄

**Comment by PeterDaveHello** [comment]
@sufuf3 see the merged/modified commit for the correct commit message format.

**Pull request merged**